src/components/formBuilder/formFields/MASFiltersField.tsx

```diff
@@ -0,0 +1,100 @@
+/* Import Dependencies */
+import { Row, Col } from 'react-bootstrap';
+import { Field } from 'formik';
+import { cloneDeep } from 'lodash';
+
+/* Import Utilities */
+import { ParseString } from 'app/Utilities';
+
+/* Import Types */
+import { Dict } from 'app/Types';
+
+/* Import Sources */
+import HarmonisedAttributes from 'sources/hamonisedAttributes.json';
+
+/* Import Components */
+import MappingFilter from './MappingFilter';
+
+
+/* Props Typing */
+interface Props {
+    name: string,
+    visibleName: string,
+    formValues?: Dict,
+    SetFieldValue?: Function
+};
+
+
+const MASFiltersField = (props: Props) => {
+    const { name, visibleName, formValues, SetFieldValue } = props;
+
+    /* Base variables */
+    const harmonisedAttributes: Dict = cloneDeep(HarmonisedAttributes);
```

Comment:
I will create a task to change this as I think the harmonised attributes don't align with the new data model.

Reply:
Nein, will remove those and change this to open text fields